.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -8,6 +8,23 @@ labels: untriaged, release, v{{ env.VERSION }}
 
 I noticed that a manifest was automatically created in [manifests/{{ env.VERSION }}](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}). Please follow the following checklist to make a release.
 
+<details><summary>How to use this issue</summary>
+<p>
+
+## This Release Issue
```

Comment:
This is from what I captured in the exiting issue, but this can be expanded greatly